### ISSUE TITLE
Stream HGTV on OSMC with Kodi 19.1 in RPi 3B+

### ISSUE THREAD

**Issue by 32Dexter** [Dis90/plugin.video.discoveryplus#63]
I can't play HGTV live stream,
this is my log:
```
2021-11-23 19:54:45.083 T:886      INFO <general>: Skipped 1 duplicate messages..
2021-11-23 19:54:45.083 T:886      INFO <general>: initializing python engine.
2021-11-23 19:54:48.862 T:886      INFO <general>: CPythonInvoker(17, /home/osmc/.kodi/addons/plugin.video.discoveryplus/addon.py): script successfully run
2021-11-23 19:54:49.644 T:886      INFO <general>: Python interpreter stopped
2021-11-23 19:54:52.303 T:895      INFO <general>: initializing python engine.
2021-11-23 19:54:56.477 T:895   WARNING <general>: Attempt to use invalid handle 8
2021-11-23 19:54:56.480 T:895      INFO <general>: CPythonInvoker(18, /home/osmc/.kodi/addons/plugin.video.discoveryplus/addon.py): script successfully run
2021-11-23 19:54:57.279 T:895      INFO <general>: Python interpreter stopped
2021-11-23 19:54:59.199 T:904      INFO <general>: initializing python engine.
2021-11-23 19:55:03.068 T:904      INFO <general>: CPythonInvoker(19, /home/osmc/.kodi/addons/plugin.video.discoveryplus/addon.py): script successfully run
2021-11-23 19:55:03.859 T:904      INFO <general>: Python interpreter stopped
2021-11-23 19:55:05.766 T:908      INFO <general>: initializing python engine.
2021-11-23 19:55:09.786 T:908   WARNING <general>: [script.module.inputstreamhelper] Widevine update check was made on 2021-11-23 14:31
2021-11-23 19:55:09.789 T:908      INFO <general>: CPythonInvoker(20, /home/osmc/.kodi/addons/plugin.video.discoveryplus/addon.py): script successfully run
2021-11-23 19:55:09.815 T:384      INFO <general>: VideoPlayer::OpenFile: plugin://plugin.video.discoveryplus/?action=play&video_id=36&video_type=channel
2021-11-23 19:55:09.995 T:918      INFO <general>: Creating InputStream
2021-11-23 19:55:10.004 T:918      INFO <general>: AddOnLog: inputstream.adaptive: SetVideoResolution (1920 x 1080)
2021-11-23 19:55:10.285 T:918      INFO <general>: AddOnLog: inputstream.adaptive: Successfully parsed manifest file. #Periods: 1, #Streams in first period: 2, Type: live, Download speed: 0.0000 Bytes/s
2021-11-23 19:55:10.659 T:908      INFO <general>: Python interpreter stopped
2021-11-23 19:55:11.662 T:918     ERROR <general>: CCurlFile::FillBuffer - Failed: HTTP returned error 400
2021-11-23 19:55:11.662 T:918     ERROR <general>: CCurlFile::Open failed with code 400 for https://discovery-eur.conax.cloud/widevine/license:                                
2021-11-23 19:55:11.662 T:918     ERROR <general>: AddOnLog: inputstream.adaptive: License server returned failure
2021-11-23 19:55:11.662 T:918     ERROR <general>: AddOnLog: inputstream.adaptive: License update not successful (no keys)
2021-11-23 19:55:11.729 T:918     ERROR <general>: AddOnLog: inputstream.adaptive: Initialize failed (SingleSampleDecrypter)
2021-11-23 19:55:11.730 T:918     ERROR <general>: CVideoPlayer::OpenInputStream - error opening [plugin://plugin.video.discoveryplus/?action=play&video_id=36&video_type=channel]
2021-11-23 19:55:11.730 T:918      INFO <general>: CVideoPlayer::OnExit()
2021-11-23 19:55:11.737 T:918      INFO <general>: ADDON: Dll Destroyed - InputStream Adaptive
2021-11-23 19:55:11.764 T:384      INFO <general>: CVideoPlayer::CloseFile()
2021-11-23 19:55:11.764 T:384      INFO <general>: VideoPlayer: waiting for threads to exit
2021-11-23 19:55:11.764 T:384      INFO <general>: VideoPlayer: finished waiting
```

**Comment by Dis90** [comment]
Have you tried to play channel from website?

**Comment by 32Dexter** [comment]
> Have you tried to play channel from website?

Sure, from the same connection it works perfectly!

**Comment by Dis90** [comment]
Seems to be some sort of problem with Widevine. Can you try with Windows or Mac?

**Comment by 32Dexter** [comment]
> Seems to be some sort of problem with Widevine. Can you try with Windows or Mac?

I took a test,
I have installed Kodi 18.9 with your addon version 1.5.15 in my pc with Windows 10 and HGTV works perfectly.
I did the same procedure, i.e. installed LibreELEC 9.2.8 (Kodi 18.9) and your addon version 1.5.15 in my Raspberry Pi 3B +, with the exact same connection and it doesn't work.
Also I tried to change O.S. with OSMC (Kodi 19.1) your addon version 1.5.24 in my Raspberry Pi 3B +, with the exact same connection and it doesn't work.
I am starting to believe that the problem is related to Widevine CDM because other addons that use Widevine also have problems.
